cumulusci/tasks/bulkdata/load.py

```diff
@@ -101,13 +105,15 @@ def _run_task(self):
             if name in self.after_steps:
                 for after_name, after_step in self.after_steps[name].items():
                     self.logger.info(f"Running post-load step: {after_name}")
-                    result = self._load_mapping(after_step)
+                    result = self._execute_step(after_step)
                     if result.status is DataOperationStatus.JOB_FAILURE:
                         raise BulkDataException(
                             f"Step {after_name} did not complete successfully: {','.join(result.job_errors)}"
                         )
 
-    def _load_mapping(self, mapping):
+    def _execute_step(
+        self, mapping: MappingStep
+    ) -> Union[DataOperationJobResult, MagicMock]:
```

Comment:
I don't like having to include mocks in the type signature. Is there no way to tell pydantic to always allow them?

Reply:
I think you mean typeguard.

And yes, there is a plan to fix it at the Typeguard level but not merged yet.

https://github.com/agronholm/typeguard/pull/105

I'll bug the maintainer again, but if he doesn't respond we might have to figure out whether to fork, abandon or continue to work around small bugs in typeguard. I found another one on the weekend. Easy to fix, but only if he accepts my PRs.